www-apps/radicale/radicale-2.1.11-r1.ebuild

```diff
@@ -1,9 +1,9 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI="7"
 
-PYTHON_COMPAT=( python3_6 )
+PYTHON_COMPAT=( python3_{6,7} )
```

Comment:
You don't need to revbump when changing PYTHON_COMPAT. We usually require "up-to-standards" ebuilds when you commit a new ebuild (hence @epsilon-0 nitpicking a lot), but for this, no new file is needed. Just edit the PYTHON_COMPAT in place.

Let me knof if you cba to update this, I'll commit it then.

Reply:
Oops, it actually already wasn’t a revbump, I just forgot to change the title and commit message. Fixed in 6b0350e0e5e145013d052e509ed9b11b1e2b4e11.

Reply:
Sorry but I still see irrelevant changes to this ebuild. **ONLY** edit `PYTHON_COMPAT`, **nothing** else.

Reply:
Sorry, I didn’t understand what you said; I thought you meant “you don’t need to revbump”, not “please do not touch anything other than PYTHON_COMPAT”. I thought all the changes I did here were minor cleanups that did not require revbump (EAPI bump, quoting a variable, and irrelevant dep changes) because they do not affect the installed files or, to a material extent, the depgraph. But if that’s what you prefer, then e87191b19479d6babe771ff31078fd5838d202d6.